event-queue/kafka/src/main/java/com/netflix/conductor/contribs/queue/kafka/KafkaObservableQueue.java

```diff
@@ -263,10 +268,31 @@ private void handleFetchedRecords(
             records.partitions()
                     .forEach(
                             partition -> {
-                                List<ConsumerRecord<String, String>> partitionRecords =
-                                        records.records(partition);
+                                List<KafkaTransactionRecord> transactionRecords = new ArrayList<>();
+
+                                records.records(partition).forEach((record) -> {
+                                    logger.info(
+                                            "Consumer Record: "
+                                                    + "key: {}, "
+                                                    + "value: {}, "
+                                                    + "partition: {}, "
+                                                    + "offset: {}",
+                                            record.key(),
+                                            record.value(),
+                                            record.partition(),
+                                            record.offset());
+                                    try {
+                                        KafkaTransactionRecord transactionRecord = new KafkaTransactionRecord(record, this.tracingProvider);
+                                        transactionRecord.start();
+
+                                        transactionRecords.add(transactionRecord);
+                                    } catch (Exception e) {
+                                        logger.error("some error", e);
```

Comment:
is there a better log message? maybe something like:

```suggestion
                                        logger.error("error creating kafka transaction record", e);
```